discord/types/snowflake.py

```diff
@@ -1,44 +1,43 @@
-from __future__ import annotations
-
-from pydantic import BaseModel
+import arrow
 
 __all__ = ("Snowflake",)
 
 
-class Snowflake(BaseModel):
```

Comment:
Why remove the entire class?
Ik in dpy it's an alias for int but we're gonna do some extra stuff.